msi/GitProduct.wxs

```diff
@@ -99,11 +99,12 @@
     <SetProperty Id="CMDLINE_GITGUIHERE" Value="[GITGUIHERE]" Before="AppSearch" Sequence="first" />
     <SetProperty Id="GITGUIHERE" Value="[CMDLINE_GITGUIHERE]" After="AppSearch" Sequence="first">CMDLINE_GITGUIHERE</SetProperty>
 
-    <Property Id="ASSOCIATEGITFILES" Value="1">
+    <SetProperty Id="CMDLINE_ASSOCIATEGITFILES" Value="[ASSOCIATEGITFILES]" Before="AppSearch" Sequence="first" />
+    <Property Id="ASSOCIATEGITFILES">
```

Comment:
I would prefer to add all these lines in this here commit, rather than adding them in a previous commit and then fixing them here...